token/token_test.go

```diff
@@ -126,7 +126,7 @@ func TestIsServiceAccountTokenInContextClaimAbsent(t *testing.T) {
 
 func TestIsServiceAccountTokenInContextClaimIncorrect(t *testing.T) {
 	tk := jwt.New(jwt.SigningMethodRS256)
-	tk.Claims.(jwt.MapClaims)["service_accountname"] = "Not-auth"
+	tk.Claims.(jwt.MapClaims)["service_accountname"] = "Not-fabric8-auth"
```

Comment:
To validate that "auth" will no longer work, you should pass in "auth" here instead of "not-fabric8-auth"